app/components/post/post.js

```diff
@@ -148,6 +149,32 @@ class Post extends PureComponent {
         });
     };
 
+    handleAddReaction = () => {
+        const {intl, navigator, post, theme} = this.props;
+
+        MaterialIcon.getImageSource('close', 20, theme.sidebarHeaderTextColor).
```

Comment:
Can you join this line to the one below it? My local ESLint complains about the indentation following this for some reason while the Jenkins one does not